core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlEvaluateResult.java

```diff
@@ -66,10 +66,20 @@ protected boolean executeAction() throws InterruptedException {
         boolean hasResult = convertHasResult(getExpectedResult().getValue());
         String connectionName = convertConnectionName(getConnectionName().getValue());
 
-        Connection connection = ConnectionConfiguration.getInstance()
-                .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+        Connection connection;
+        try {
+            connection = ConnectionConfiguration.getInstance()
+                    .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
+                    .get();
+        } catch (Exception e) {
+            throw new RuntimeException("Unknown connection name");
+        }
+
         Database database = DatabaseHandler.getInstance().getDatabase(connection);
+        if (database == null) {
+            throw new RuntimeException("Error establishing DB connection");
+        }
+
```

Comment:
DatabaseHandler.getInstance().getDatabase(...) will never return a null object. It will either return a Database object or throw an Exception. There should be no need to check parameter for null

Reply:
removed